Terminal.Gui/Views/CollectionNavigation/CollectionNavigatorBase.cs

```diff
@@ -61,21 +37,21 @@ public int GetNextMatchingItem (int currentIndex, char keyStruck)
             var candidateState = "";
             var elapsedTime = DateTime.Now - _lastKeystroke;
 
-            Logging.Trace($"CollectionNavigator began processing '{keyStruck}', it has been {elapsedTime} since last keystroke");
+            Logging.Trace ($"CollectionNavigator began processing '{keyStruck}', it has been {elapsedTime} since last keystroke");
```

Comment:
Change these to `Debug`, please. IMO `Trace` should only be used for very-core, low-level things. No?

Reply:
Sure, I've changed to Debug in [ebecbc8](https://github.com/gui-cs/Terminal.Gui/pull/4029/commits/ebecbc8dfb1021273a6ea10ca8a7f0d077d00f9e)